components/script/dom/htmlscriptelement.rs

```diff
@@ -1064,6 +1080,11 @@ impl HTMLScriptElement {
             Ok(script) => script,
         };
 
+        if script.type_ == ScriptType::ImportMap && script.import_map_result.is_none() {
+            self.dispatch_error_event(can_gc);
+            return;
+        }
```

Comment:
We are doing this check because we are using `ScriptOrigin` to pass the import map result. But the current implementation guarantees that `import_map_result` must exist if the type is `ImportMap`